src/reducers/handleButtonClick.js

```diff
@@ -9,6 +9,7 @@ const handleButtonClick = (state = initialButton, action) => {
     case HANDLE_CREATE:
       const { isCreateButtonClicked } = action.payload;
       return {
+        ...state,
         isCreateButtonClicked: isCreateButtonClicked,
```

Comment:
see https://eslint.org/docs/latest/rules/object-shorthand

```suggestion
        isCreateButtonClicked,
```